### PR TITLE
DEP Upgrade to symfony 7

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/296

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Please also remove the `symfony/process` conflict.

**Comment by GuySartorelli** [comment]
@emteknetnz Please make this change

File: tests/php/View/SSViewerTest.php [hunk 1]
```diff
@@ -679,7 +679,7 @@ public function testLoopWhitespace()
 $ItemOnItsOwnLine
 <% end_loop %>
 after',
-$data
+                $data
```
Comment (emteknetnz) [inline tests/php/View/SSViewerTest.php:682]: This was to fix a PHPCS issue

**Comment by emteknetnz** [comment]
Have removed symfony/process conflict

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**